opencog/benchmark/diary.txt

```diff
@@ -1063,6 +1065,18 @@ How fast is std::vector<Handle>, really?
   emplace_back      7.26         3.35        1.96         1.87
   reserve           7.66         6.01        4.75         4.13
 
+count: the number of handles that are pushed/emplaced back.
+
+push_back: call the push-back method
+     example: HandleSeq oset; oset.push_back(ha); oset.push_back(hb);
+
+emplace-back: call the emplace_back method
+
+reserve: create vector of fixed size, then assign directly.
+     example: HandleSeq oset(2); oset[0] = ha; oset[1] = hb;
+
+Looks like direct assignment is a winner!
```

Comment:
Did you try reserve of fixed size then emplace_back? I think that might save a copy.